### PR TITLE
chore(sdk): 2.0.0-alpha.4 release

### PR DESCRIPTION
**Pull request by chongyouquan**
**Description of your changes:**

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: sdk/RELEASE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Current Version (Still in Development)
```
Comment (chensun) [inline sdk/RELEASE.md:1]: Copy the template on top of this and leave it blank, then change this to `2.0.0-alpha.4`.

**Review by chensun — CHANGES_REQUESTED** [review]

File: sdk/RELEASE.md (hunk 1 above)
Comment (chongyouquan) [reply sdk/RELEASE.md:1]: Done.

File: sdk/RELEASE.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+### For Component Authors
+
+## Deprecations
+
+## Bug Fixes and Other Changes
+
+## Documentation Updates
+
+# Current Version (2.0.0-alpha.4)
```
Comment (chensun) [inline sdk/RELEASE.md:17]: nit: Just keep `2.0.0-alpha.4` (remove `Current Version` and the parentheses)

Comment (chensun) [reply sdk/RELEASE.md:17]: Can you help fix `Current Version (2.0.0-alpha.3)` as well? to be consistent with all previous versions.

Comment (chongyouquan) [reply sdk/RELEASE.md]: Ok sure. Thanks for pointing these out! Done.

**Review by chensun — APPROVED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**